### DISCUSSION TITLE
Disable Backoffice Login Setting

### DISCUSSION THREAD

**Discussion by garpunkal** [umbraco/Umbraco-CMS#12754] [Features and ideas]
The ability to set a global configuration setting to disable the backoffice login. 

This is important to prevent users logging into content delivery servers instances of Umbraco. 

This is instead of using IIS redirects to prevent login.

**Comment by mikaellindemann** [comment]
With Umbraco 9 and later it is possible to disable the backoffice by not calling `.UseBackOffice()` in the middleware pipeline and `.WithBackOfficeEndpoints()` in the endpoints section. In Umbraco 9 those calls would be in the `Startup.Configure(...)` method.
